config/locales/en.yml

```
@@ -216,21 +216,21 @@ en:
-          date_of_birth: Use hint text to help people answer the question. For a date of birth question you could use ‘For example, 27 3 1998’.
-          default: Use hint text to help people answer the question. For example, to explain the format the answer should be in, or where to find the information you’ve asked for.
-          email: You could use hint text to tell people how you’ll use their email address. For example, ‘We’ll only use your email address to contact you about your application.’
-          long_text: Use hint text to help people answer the question. For example, you could give a bit more detail about the information you need.
-          name: Use hint text to help people answer the question. For example, you might need to ask people to enter their name as it’s written on an official document such as a passport or driving licence.
-          national_insurance_number: Use hint text to help people answer the question. For example, ‘It’s on your National Insurance card, benefit letter, payslip or P60. For example, QQ 12 34 56 C.’
-          number: Use hint text to help people answer the question. For example, ‘Do not include bank holidays’.
-          organisation_name: Use hint text to help people answer the question. For example, you might need to ask people to enter the registered name of their company.
-          other_date: Use hint text to help people answer the question. For a date question you could use ‘For example, 27 3 2007’.
-          phone_number: Use hint text to help people answer the question. For example, ‘You can provide either a home or mobile phone number.’
-          radio: Use hint text to help people answer the question. For a question where people can only select one answer you might want to use ‘Select one option’.
-          single_line: Use hint text to help people answer the question. For example, you could say what format the answer should be in or where to find it.
+          address: You can add a short hint to help people answer the question. For example, ‘We’ll send your licence to this address.’
+          checkbox: You can add a short hint to help people answer the question. For a question where people can select more than one answer you might want to use ‘Select all that apply’.
+          date: You can add a short hint to help people answer the question. For a date question you could use ‘For example, 27 3 2007’.
+          date_of_birth: You can add a short hint to help people answer the question. For a date of birth question you could use ‘For example, 27 3 1998’.
+          default: You can add a short hint to help people answer the question. For example, to explain the format the answer should be in, or where to find the information you’ve asked for.
+          email: You could add a short hint to tell people how you'll use their email address. For example, ‘We’ll only use your email address to contact you about your application.’
```

I know it's not really related to the change your making but I wonder if we should follow the same pattern we used for the address hint for this one?

```suggestion
          email: You can add a short hint to help people answer the question. For example, you could tell people how you'll use their email address.
```

---

Could do, yes, but I was thinking that this example text is quite useful and people might want to copy and paste it (more so than the address example we had).

---

Yeah possibly true. I think I'm more concerned that this hint is quite suggestive of one way to use the hint text and not as broad as the others that start with "You can add a short hint to help people answer the question". This was an existing issue with the hint though and not related to this update - so fine to stick with what you have. :)

---

Yeah, I agree it's suggestive, but it feels like it's usefully suggestive... I've made a note in the hint text doc that this is something to think about though.